static/json/changelog.json

```diff
@@ -78,5 +78,10 @@
         "date": 1544850000000,
         "title": "Synced URLs with entire CrUX corpus",
         "desc": ""
+    },
+    {
+        "date": 1596254400000,
+        "title": "Faster mobile CPU throttling",
+        "desc": "WebPageTest changed its mobile CPU throttling from cgroups to Chrome's built-in throttling"
```

Comment:
Maybe be a bit more descriptive: "WebPageTest changed its mobile CPU throttling from cgroups to Chrome's built-in throttling. The cgroups had more overhead than expected when running in VM's and switching to Chrome's built-in throttling eliminated the extra overhead and resulted in more accurate throttling (which led to slightly faster performance)."

Reply:
Thanks! Also added a link to the issue for more context.